.eslintrc.json

```
@@ -25,7 +25,7 @@
   "rules": {
     "comma-spacing": ["error", { "before": false, "after": true }],
     "cypress/no-pause": "error",
-    "indent": ["error", 2],
+    "@typescript-eslint/indent": ["error", 2],
```

I'll do that in another PR
```suggestion
     "indent": ["error", 2],
```